### PR TITLE
Split out browser-only entry point

### PR DESCRIPTION
**Pull request by lencioni**
This allows us to take advantage of
https://github.com/airbnb/js-shims/pull/4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,5 +20,17 @@ Included shims:
...
+## Only browser shims
+
+If you only want to bring in the browser shims and not the JS language shims
+(from `airbnb-js-shims`), you can import `airbnb-browser-shims/browser-only`. If
+you choose this route, you will want to be sure that you are properly bringing
+in the language shims for the browsers you support somehow. For example:
+
+```js
+import 'airbnb-browser-shims/browser-only';
+import 'airbnb-js-shims/target/es2015';
```
Comment (ljharb) [inline README.md]: the JS shims should always come first

**Review by ljharb — CHANGES_REQUESTED** [review]
LGTM pending one docs change

File: README.md (hunk 1 above)
Comment (lencioni) [reply README.md]: Whoops! Of course. Fixed now.

**Review by ljharb — APPROVED** [review]

**Comment by lencioni** [comment]
Sheesh, Travis is being slow this morning. Since the push build passed and my last revision only modified the readme, I'm going to merge this now.

**Pull request merged**